types/masonry-layout/index.d.ts

```diff
@@ -51,10 +56,13 @@ declare namespace Masonry {
         originLeft?: boolean;
         originTop?: boolean;
         horizontalOrder?: boolean;
+        hiddenStyle?: hiddenOrVisibleStyleOption;
+        visibleStyle?: hiddenOrVisibleStyleOption;
 
         // setup
         containerStyle?: {};
         transitionDuration?: any;
+        stagger?: string | number;
```

Comment:
just question, if you want to accept nulls:
https://github.com/desandro/masonry/blob/master/dist/masonry.pkgd.js#L1684

Reply:
Just noticed I forgot to add the docs link for this:
https://masonry.desandro.com/options.html#stagger

The above link describes this option. There also is the following sentence on top of the page:
> All options are optional, but columnWidth and itemSelector are recommended.